pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -155,7 +157,43 @@ func flowStringToAntreaConnection(flow string, zoneFilter uint16) (*flowexporter
 			} else {
 				conn.TupleReply.DestinationPort = uint16(val)
 			}
-		} else if strings.Contains(fs, "zone") {
+		case strings.Contains(fs, "packets"):
+			fields := strings.Split(fs, "=")
+			val, err := strconv.Atoi(fields[len(fields)-1])
+			if err != nil {
+				return nil, fmt.Errorf("conversion of packets %s to int failed", fields[len(fields)-1])
+			}
+			if !isReply {
+				conn.OriginalPackets = uint64(val)
+			} else {
+				conn.ReversePackets = uint64(val)
+			}
+		case strings.Contains(fs, "bytes"):
+			fs = strings.TrimSuffix(fs, ")")
+			fields := strings.Split(fs, "=")
+			val, err := strconv.Atoi(fields[len(fields)-1])
+			if err != nil {
+				return nil, fmt.Errorf("conversion of bytes %s to int failed", fields[len(fields)-1])
+			}
+			if !isReply {
+				conn.OriginalBytes = uint64(val)
+			} else {
+				conn.ReverseBytes = uint64(val)
+			}
+		case strings.Contains(fs, "start"):
```

Comment:
Do we not see the StopTime in the output of dump conntrack flows?

Reply:
I didn't see stoptime related string in the cmd output. I saw the comments corresponding to StopTime in Connection struct:
```
	// For invalid and closed connections: StopTime is the time when connection was updated last.
	// For established connections: StopTime is latest time when it was polled.
```
Does it mean StopTime is not present in the output of dump conntrack flow and we need to handle it by ourselves?

Reply:
That was just an interpretation of stop time. May be doing a bit of experimentation with established and closed connections can help. If stoptime is there in closed connections, then it would be better to have that field when parsing flow strings.

Reply:
Please add a todo for stoptime to revisit later as discussed offline.